docs/spark/how-to-guides/connect-to-sql-server.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Connect to SQL server locally from .NET for Apache Spark
+description: Learn how to connect to an SQL server instance from your .NET for Apache Spark application.
+ms.date: 09/24/2020
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Connect to SQL server locally from .NET for Apache Spark
+
+In this article, you learn how to connect to an SQL server instance from your [.NET for Apache Spark](https://github.com/dotnet/spark) application.
```

Comment:
```suggestion
In this article, you learn how to connect to a SQL server instance from your [.NET for Apache Spark](https://github.com/dotnet/spark) application.
```

Reply:
I think `an` is the correct usage in this case.

Reply:
@Niharikadutta Re: a/an I'd say it out loud and go with what sounds better. For me, a SQL server instance sounds more natural than an SQL server instance.

I also made a minor tweak to point to docs instead of the repo.